### PR TITLE
Load picturefill with polyfill.io

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

We currently have an old forked version of [picturefill](https://github.com/scottjehl/picturefill) (a `HTMLPictureElement` polyfill) checked into our codebase. Since the [browser support](http://caniuse.com/#search=picture) for `HTMLPictureElement` is pretty high, we should not be serving this to all user agents.

This PR deletes the picturefill fork from our codebase, and instead conditionally loads the vanilla version of picturefill for appropriate user agents using polyfill.io.

Since the API for picturefill has improved, we no longer need to call the global picturefill function explicitly, and we can rely on the polyfill being applied to `HTMLPictureElement`s already in the DOM, or dynamically added to the DOM.

## What is the value of this and can you measure success?

- Less code to maintain
- Seamless `HTMLPictureElement` support
- Less code sent to up-to-date browsers

### Any downside?

- Our fork of picturefill used fastdom to batch reads from / updates to the DOM. Older browsers will be losing this performance optimisation when we switch to the vanilla version of picturefill.
- polyfillio.fallback.js has increased from 182KB to 260KB (unminified / un-gzipped)

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Screenshots

No

## Tested in CODE?

YES!

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
Great!

**Review by NataliaLKB — APPROVED** [review]
This is a big PR, but looks really good! Are we able to test this in CODE at all before PROD?

File: static/src/javascripts/bootstraps/standard/main.js [hunk 1]
```diff
@@ -185,10 +184,6 @@ const bootStandard = (): void => {
         removeCookie('GU_AF1');
     }
 
-    // Images
-    upgradePictures();
-    listen();
```
Comment (gustavpursche) [inline static/src/javascripts/bootstraps/standard/main.js]: I guess the definition of `listen()` could be removed too?

**Review by gu-stav — APPROVED** [review]
Just a minor idea - the rest looks great to me. I agree with Natalia: we should deploy this on code and then click around in the various places with old browsers ... Let me know, if you want any help with this.

File: static/src/javascripts/bootstraps/standard/main.js (hunk 1 above)
Comment (SiAdcock) [inline static/src/javascripts/bootstraps/standard/main.js]: I've removed the whole `common.modules/ui/images` module!

**Pull request merged**